jobs/Backend/Task/ExchangeRateProvider/ExchangeRate.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace ExchangeRateProvider
+{
+    public class ExchangeRate
+    {
+        public ExchangeRate(int amount, CurrencyCode sourceCurrency, CurrencyCode targetCurrency, decimal rate)
+        {
+            Amount = amount;
+            SourceCurrency = sourceCurrency;
+            TargetCurrency = targetCurrency;
+            Rate = rate;
+        }
+
+        public int Amount { get; }
+
+        public CurrencyCode SourceCurrency { get; }
+
+        public CurrencyCode TargetCurrency { get; }
+
+        public decimal Rate { get; }
+
+        public override string ToString()
+        {
+            return $"{Amount} {SourceCurrency} = {Rate} {TargetCurrency}";
+        }
+
+        public override bool Equals(object obj)
```

Comment:
I needed to override Equals and GetHashCode methods in order to be able to compare objects of this instance for a testing purposes.